testsuite/gnat2goto/testsuite.py

```diff
@@ -52,25 +53,39 @@ def main():
     if m.options.discs:
         discs += m.options.discs.split(',')
 
-    test_metrics = {'total': len(test_list)}
+    test_metrics = {'total': len(test_list), 'uok': 0}
 
-    collect_result = generate_collect_result(
+    # Generate a standard 'collect_result' function...
+    generated_collect_result = generate_collect_result(
         result_dir = m.options.output_dir,
         results_file = m.options.results_file,
         output_diff = m.options.view_diffs,
         metrics = test_metrics)
 
+    # ... and then wrap that generated 'collect_result' function in something
+    # that will also accumulate 'UOK' test results
+    def collect_result_including_uok(name, process, _job_info):
+        generated_collect_result(name, process, _job_info)
+        test_name = test_name = os.path.basename(name)
```

Comment:
Am I missing something?

Reply:
looks like a typo

Reply:
Or some python magic, you never know.

Reply:
Argh, yes - typo, and I didn't spot the comments here before merging - my bad. I'll do another PR to fix.